src/fu-self-test.c

```diff
@@ -2194,7 +2194,7 @@ fu_device_list_compatible_func (gconstpointer user_data)
 	g_assert_cmpint (changed_cnt, ==, 1);
 
 	/* device2 should inherit the vendor ID and version from device1 */
-	g_assert_cmpstr (fu_device_get_vendor_id (device2), ==, "USB:0x20A0");
+	g_assert_true (fu_device_has_vendor_id (device2, "USB:0x20A0"));
```

Comment:
I think the self tests need to add some new corner cases for multiple vendor IDs and how the interactions with them work.

Reply:
I've added in one on either side.